leapp/libraries/stdlib/__init__.py

```diff
@@ -0,0 +1,25 @@
+"""
+:py:mod:`leapp.libraries.stdlib` represents a location for functions that otherwise would be defined multiple times across leapp actors
+ and at the same time are really useful for other actors.
+
+Example:
+    If the function call() is called from an actor
+
+      from leapp.libraries.stdlib import call
+
+"""
+import six
+import subprocess
+import os
+
+def call(args, split=True):
+    ''' Call external processes with some additional sugar '''
```

Comment:
Please document this function properly 'With some additional sugar' isn't really a  good documentation, plus I am missing parameter description:
```python
    """
    Calls an external program and captures it ouput. Additionally the output is automatically utf-8 decoded, and output to stderr is suppressed and redirected to /dev/null.

    :param args: Command to execute
    :type args: list
    :param split: Split the output on newlines?
    :type split: bool
    :return: stdout output, 'utf-8' decoded, split by lines if split=True
    :rtype: unicode/str or [unicode/str] if split=True
   """
```

Also I would call this function rather 'check_output' since this is what it is using and gives a bit of a background to people using this function on what this actually is supposed to be doing. 
The name `call` is misleading IMHO, this was fine for private libraries but stdlib functions need to be named better.

Reply:
I would extend the info, that we talked with @shaded-enmity about the extensions of the function, so check_output doesn't fit as well because it suggests it should be more like known `subprocess.check_output` function. However it is mentioned to do more things and `call` is generic name, which sounds like something what I will find first. So I would go with call even. Anyway, I can go even with the check_output function, as we can inside call the generic `call` function with expected params.